astroquery/jplspec/core.py

```diff
@@ -156,6 +156,9 @@ def _parse_result(self, response, verbose=False):
         QN":   Quantum numbers for the lower state.
         """
 
+        if 'Zero lines were found' in response.text:
+            raise ValueError(f"Response was empty; message was '{response.text}'.")
```

Comment:
We have a bunch of more specific errors, if you think that a new class would be useful to have for all the astroquery  linelist modules to be raised, this is  a good time to introduce it. (I don't have a user workflow lined out for these modules so have no idea whether the extra info in a "LinenotFoundError" or similar is useful or not, but feel that `ValueError` maybe a bit too generic)

Reply:
Yes, I don't think it's an `InvalidQueryError` as the query can be still technically valid if there are no lines in that region. Subclassing ValueError sounds great I like`EmptyResponseError(ValueError)`. But again, if you say that this would not provide any useful addition to the users about the queries, then the good old `ValueError` is fine with me, too (e.g would you think they try/except for this error in a workflow? If yes, then I think it's better to keep it more specific than VE, but if it's nonsense to try/except then we don't need the new class)

Reply:
I added the `EmptyResponseError` and modified the earlier Exception too (it is genuinely "invalid query")